app/steps/ui/signout/index.js

```diff
@@ -0,0 +1,29 @@
+const Step = require('app/core/steps/Step');
+const config = require('app/config');
+const logger = require('app/components/logger')('Init');
+const services = require('app/components/services');
+const SECURITY_COOKIE = '__auth-token-' + config.payloadVersion;
```

Comment:
Can we use a template literal here instead of concatenating the strings? For example:

```
const SECURITY_COOKIE = `__auth-token-${config.payloadVersion}`;
```